app/views/cancelation_mailer/cancelation_confirmation.text.erb

```diff
@@ -4,3 +4,5 @@
 
 <%= t("mailer.cancelation_mailer.cta") %>
 <%= proposal_url(@proposal) %>
+
+<%= t("mailer.footer", feedback_url: feedback_url) %>
```

Comment:
Just to make sure - This is a template that needs a second pass right? The footer is a part of the actual base template again.